src/test/ui/lint/rfc-2383-lint-reason/expect_enabled_feature_gate.rs

```diff
@@ -0,0 +1,9 @@
+// check-pass
+
+#![feature(lint_reasons)]
+
```

Comment:
What happens if you `#[expect(unfulfilled_lint_expectation)]`? Can you add a test for that?

Reply:
There is a test like this in [`expecting_missed_expectations.rs`](https://github.com/rust-lang/rust/blob/5e176b09f9c91098dff7ff475bb37bdc18320bd7/src/test/ui/lint/rfc-2383-lint-reason/expecting_missed_expectations.rs#L14-L19). Does that cover your request? :upside_down_face:

Reply:
Oh oops, sorry 😅. I don't see anything to add 👍